essential-guide/filtering.md

```diff
@@ -0,0 +1,76 @@
+# Filtering
+
+Mews POS API supports filtering using the JSON:API reserved `filter` query parameter. The `filter` query parameter can be used to filter resources by attributes.
+
+Please note that support for filters is endpoint-specific. Visit the documentation of an endpoint to get more details about the supported filters.
+
+It is recommended to always URI-encode the values of the filter query parameters as filters can contain unsafe URL characters.
```

Comment:
I recommend a breakout box here; also switch to 'active voice'.
```suggestion
> **Unsafe characters**: We recommend you always URI-encode the values of filter query parameters, because filters can contain unsafe URL characters.
```